### PR TITLE
about page with discord embed created; works with light dark toggle and mobile

### LINKED ISSUE #39: Create About Page
Implement this slight redesign of about page on the new portal (desktop and mobile friendly).

You can probably reference the code for this page on the old portal.

image
[link to figma](https://www.figma.com/file/WCc4V3sy05iF55BRum71Mv/Portal-UI-v2-Pages?type=design&node-id=22-824&t=lE8Xy66AHrvLfESP-0)

### PR DESCRIPTION
**Pull request by adahe8**
# Info

Closes **#39**, creating the about page.

# Description

This is the NextJS implementation of the about page for the ACM portal; I've added the discord embed and transferred the socials and descriptions over.

## Changes

Added discord embed, used currentColor CSS var to make the svg icons responsive to the toggle, replaced and updated some svg icons to slightly higher quality ones.

# Type of Change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
      linting/formatting)
- [ ] Continuous Integration Change (Related to deployment steps or continuous integration
      workflows)
- [ ] Other: (Fill In) 

# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally on Desktop.
- [x] locally on mobile - use https://ngrok.io to get a copy on a mobile device
- [x] on the live deployment preview on Desktop.
- [x] on the live deployment preview on Mobile.
- [ ] I have run and passed all new and existing Cypress tests. Add screenshots below.

# Checklist

- [x] I have performed a self-review of my own code.
- [x] I have followed the style guidelines of this project.
- [x] I have documented my code's `src/lib` functions and commented hard to understand areas
      anywhere else.
- [x] My changes produce no new warnings.

# Screenshots

Please include a screenshot of your Cypress testing suite passing successfully.

If you made any visual changes to the website, please include relevant screenshots below.

### CODE REVIEW

File: src/lib/config.ts
Comment (raymosun) [inline src/lib/config.ts]: Since this description is only used on one page, it's better to keep it with the About page's files.

File: src/pages/about.tsx
Comment (raymosun) [inline src/pages/about.tsx]: Nit: maybe name this something like `AcmLogo` to keep it unambiguous and in line with other icon names

File: src/styles/pages/about.module.scss
Comment (raymosun) [inline src/styles/pages/about.module.scss]: In `vars.scss` we have some breakpoint size variables that could be used here. They help keep the site consistent across pages.

File: src/styles/pages/about.module.scss
Comment (raymosun) [inline src/styles/pages/about.module.scss]: `visibility: hidden` makes the discord preview invisible, but it still takes up space so it's making the page like randomly long on mobile right now. You can try using `display: none` instead.

File: src/styles/pages/about.module.scss
Comment (raymosun) [inline src/styles/pages/about.module.scss]: On this repo we try to use 'rem' instead of 'px' whenever possible for accessibility reasons. I noticed you used 'rem' in most other places so lmk if there was some reason for using 'px' here

File: src/styles/pages/about.module.scss
Comment (raymosun) [inline src/styles/pages/about.module.scss]: Smallest ever nit: you could probably replace this with `margin-left: 20px` 🤓

**Review by raymosun — CHANGES_REQUESTED** [review]
Hey, nice work on this! The discord embed fits in really nicely on this page.

I left a couple comments for you on code. Additionally,
1. Can you center the content on the page, probably by putting all the content in a container with a max-width and `margin: 0 auto`. Maybe you can reference Leaderboard, the only other completed page so we can have consistent styling for this.
2. Small thing, but I noticed the corners of the discord embed inside the border radius remain white in dark mode. Could you look into having this match the theme as well please :) 
image

File: src/styles/pages/about.module.scss
Comment (adahe8) [reply src/styles/pages/about.module.scss]: fixed! switched to rem as well

File: src/styles/pages/about.module.scss
Comment (adahe8) [reply src/styles/pages/about.module.scss]: noted and fixed!

**Review by raymosun — APPROVED** [review]

File: src/pages/about.tsx
Comment (farisashai) [inline src/pages/about.tsx:17]: No need to keep this in a variable but not a big deal either way.

File: src/pages/about.tsx
Comment (farisashai) [inline src/pages/about.tsx]: Is this div layer necessary or can you just put the className on the <a> tags directly?

File: src/pages/about.tsx
Comment (farisashai) [inline src/pages/about.tsx]: Is there a design reason to write acmurl here instead of acmucsd.com? Could be confusing and harder to find access to the main website possibly.

File: src/pages/about.tsx
Comment (farisashai) [inline src/pages/about.tsx]: I think `acmurl.com/facebook` works better here and for instagram for consistency with the discord link and because it allows the link to be updated freely through acmurl without a code commit needed. Also slightly easier to read and more memorable.

File: src/styles/pages/about.module.scss
Comment (farisashai) [inline src/styles/pages/about.module.scss]: nit: prefer absolute imports if possible for imports from outside the same directory folder in case things end up moving around

File: src/styles/pages/about.module.scss
Comment (farisashai) [inline src/styles/pages/about.module.scss]: nit: all css values like this should have units attached for clarity and no ambiguity. units here would be `px` for consistency with how the <Image> component behaves.

File: src/styles/pages/about.module.scss
Comment (farisashai) [inline src/styles/pages/about.module.scss]:
```suggestion
  .discordPreview {
```
nitpick: camelcase CSS classes

File: src/styles/pages/about.module.scss
Comment (farisashai) [inline src/styles/pages/about.module.scss:13]: Screenshot 2023-08-17 at 10 46 19 PM
```suggestion
.header {
padding: 0 2rem;
}
```
nit: can we add the 2rem padding just for the sides to align the image left side with the text content horizontally?

**Review by farisashai — APPROVED** [review]
I'm going to approve right now to not block the PR completely because the page implementation looks really good. Just some minor code nitpicks to fix though before you merge

File: src/pages/about.tsx
Comment (adahe8) [reply src/pages/about.tsx]: right you are!

File: src/pages/about.tsx
Comment (adahe8) [reply src/pages/about.tsx]: nope! I was just copying the links over from the old portal - fixed

**Pull request merged**
